index.html

```diff
@@ -80,6 +80,7 @@
 
 </head>
 <body class="fixedheight">
+<script>if (typeof module === 'object') { window.module = module; module = undefined; }</script>

```

Comment:
Let's do:

    <script>if (typeof module === 'object') { window.module = module; module = undefined; }</script> // to make it work with Electron https://github.com/electron/electron`

Reply:
Sounds good